scipy/stats/tests/test_mstats_extras.py

```diff
@@ -139,6 +139,9 @@ def test_hdquantiles_sd(self):
         # Test actual values for good measure
         assert_almost_equal(hd_std_errs, [0.0379258, 0.0380656, 0.0380013])
 
+        two_data_points = ms.hdquantiles_sd([1, 2])
+        assert_almost_equal(two_data_points, [0.5, 0.5, 0.5])
```

Comment:
Why is this being added? Is this a corner case that the code in `main` or an early version of this PR had trouble with, or is it unrelated to the changes here?

Reply:
It's pretty much unrelated; I just wanted to make sure it worked with 2 data points.